Nodejs/Product/Nodejs/visualstudio_nodejs_repl.js

```diff
@@ -92,10 +79,6 @@ context.require = require;
 context.module = module;
 context = vm.createContext(context);
 
-util.inspect.styles['number'] = 'blue';
-util.inspect.styles['boolean'] = 'blue';
-util.inspect.colors['blue'] = [94, 39];
-
```

Comment:
This made the default for the `number` and `Boolean` blue... which was harder to get right in all the themes. Decided to stick to the node default of Yellow.